app/models/course/assessment.rb

```diff
@@ -86,6 +86,16 @@ class Course::Assessment < ApplicationRecord
     end
   end)
 
+  # Used by the with_actable_types scope in Course::LessonPlan::Item.
+  # Edit this to remove items for showing in the lesson plan.
+  #
+  # Here, actable_data contains the list of tab IDs to be removed.
+  scope :showable_in_lesson_plan, (lambda do |actable_data|
```

Comment:
Since we are returning ids not instances of `Assessment`s here, should we call this `:ids_showable_in_lesson_plan`?

My bad. Should have been more accurate earlier.